### PR TITLE
UE5 Addition

### PR DESCRIPTION
**Pull request by VerzatileDev**
Changes
---

Fixed an ( Issue ) Related to Char* , from illegal move to a string value instead

This ensures and avoids the issue of rebuilding it through the batch file
Additionally fixes the Problem for manual build through Project - > Plugins
( Note: Building is only nessecary if the files such as " Binaries / Intermediate " are missing from the plugin files )

Updated the CLI version, which is now directly downloading the latest available on WakaTime website and supports the " waka_ " prefix before the apiKey for reference

(waka_apiKey)
(Note: CLI and Prefix use is in reference to the CLI version being used, both with or without a prefix should work based on testing, unless the Wakatime APIKey does not get changed, on that note Restart the Engine And it should work as intended sending a heartbeat to the servers).

Removed user specific code from the files

.Vscode
(Not nessecary to be included because these are built by and for each developer and commonly cannot be used by others, its directly for the one that made it)

Build Info
---

 To Download the Source, building can only be done through:

   - Batch file of Unreal Engine.
   - Add the Plugin to the Project -> Plugins ( Building it from Engine -> plugins wont work ).

(Note if there are issues Building it, please references to the following https://www.youtube.com/watch?v=2HHJ6AYnwFw&list=WL&index=428 ) And References that in  https://github.com/TheAshenWolf/WakaTimeForUE/issues ), Any issues with the code will be written in the CMD files, that can be fixed ( It sections them out in detail! )

To Download the Working version, Pre-Built for Unreal Engine 5 use or see https://github.com/VerzatileDev/WakaTimeForUE/releases/tag/v.1.2.5

Future
---

Updates to accomidate the Plugin version within code will be done in a different Update instead as the following is specifically used to build and Update the version of the plugin to accomidate UE5 and new Updates to CLI versions of Wakatime.

This includes rest of the code changes, as the following update it aimed towards fixing current problems to make it work within UE5

### CODE REVIEW

File: Source/WakaTimeForUE/Private/WakaTimeForUE.cpp [hunk 1]
```diff
@@ -429,7 +426,7 @@ void FWakaTimeForUEModule::SendHeartbeat(bool bFileSave, string FilePath, string
 	Command += "--project \"" + GetProjectName() + "\" ";
 	Command += "--entity-type \"app\" ";
 	Command += "--language \"Unreal Engine\" ";
-	Command += "--plugin \"unreal-wakatime/1.2.2\" ";
+	Command += "--plugin \"unreal-wakatime/1.2.5\" "; // Update this with the plugin version from .uplugin (Unreal Plugin Settings) (Avoid Hardcoding it here) TODO
```
Comment (VerzatileDev) [inline Source/WakaTimeForUE/Private/WakaTimeForUE.cpp:429]: See this section for Future Update to get it from the .uplugin file instead.

**Comment by VerzatileDev** [comment]
Possibly be able to change the Version that is shown on the Plugin as referenced here https://github.com/TheAshenWolf/WakaTimeForUE/issues/38 Would be a good addition so that Unreal 4 and 5 are seperated.

**Comment by TheAshenWolf** [comment]
Downloaded source, built it _(after struggles with missing SDKs)_, got few heartbeats. 

Seems to be working properly, great job!

**Comment by VerzatileDev** [comment]
Yeah I think as far as SDK's goes that is not something we have to touch nessecarily, as if they are already developing with unreal they should be familiar with it I would say at least.

**Comment by TheAshenWolf** [comment]
Oh, the SDKs were unrelated to the plugin :D I was reinstalling Unreal from scratch, so some stuff was missing.

**Pull request merged**